### ISSUE TITLE
Cannot add /weekly and /monthly to the Telegram bot keyboard

### ISSUE THREAD

**Issue by Merinorus** [freqtrade/freqtrade#6019]
Following #5527 

## Describe your environment

  * Operating system: Synology 6.2 & Docker
  * Freqtrade Version: 2021.11 running in Docker (Dockerfile based on latest stable version)
  

## Describe the problem:

I cannot add /weekly and /monthly commands to the Telegram keyboard.
I think it should be an easy fix! I could work on it since I added these two commands myself, or someone else can contribute to this "good first issue".

### Steps to reproduce:

  1. Modify the config.json file to add `/weekly` and/or `/monthly` commands
  2. Reload the bot
  
### Observed Results:

The bot will simply crash and log that /weekly and /monthly are invalid commands.

### Relevant code exceptions or logs

  ```
2021-11-30 23:23:30,308 - freqtrade.commands.trade_commands - ERROR - config.telegram.keyboard: Invalid commands for custom Telegram keyboard: ['/weekly', '/monthly']  ```

**Comment by Theagainmen** [comment]
Not sure if I can help you, but post the relevant edited code part already here so people can help you better.

**Comment by xmatthias** [comment]
This is/was an easy fix - and was fixed in #5982, and is already fixed in the develop branch.

Thanks for reporting, anyway :+1:

**Comment by Merinorus** [comment]
Ok great! That was quick. Sorry for the duplicate!
